### PR TITLE
CB-11919. Support multi region CP for databus clients

### PR DESCRIPTION
**Pull request by oleewere**
details:
- add new configuration (that should be passed in cloud-services-infra + dps-k8s) -> as thunderhad-sigmadbus is part of CSI, pass the s3 bucket name for dbus.
- use env region option for fluentd dbus plugin, if region is not used (or us-west-1 is passed), it will use the old dbus api, otherwise it will use the new /api/v1/dbus one
- update Uts

### CODE REVIEW

**Comment by lnardai** [comment]
@lturcsanyi Please work with @oleewere and get this merged, since it's needed for multi region control plane.

File: core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/diagnostics/DiagnosticsFlowService.java [hunk 1]
```diff
@@ -56,6 +56,8 @@
 
     private static final String AWS_METADATA_SERVER_V2_SUPPORT_VERSION = "0.4.9";
 
+    private static final String MULTI_AZ_SUPPORT_VERSION = "0.4.12";
```
Comment (lturcsanyi) [inline core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/diagnostics/DiagnosticsFlowService.java:59]: Is it MULTI_AZ_SUPPORT_VERSION or it should be MULTI_REGION_CP_VERSION?

**Comment by lturcsanyi** [comment]
This is an old PR, have you tested it recently?

**Comment by oleewere** [comment]
> This is an old PR, have you tested it recently?
it was done today, it seems OK

**Review by lturcsanyi — APPROVED** [review]

**Pull request merged**